### ISSUE TITLE
Can not import xgboost in Anaconda Jupyter Notebook, with error 127

### ISSUE THREAD

**Issue by wangjiahong666** [dmlc/xgboost#1644]
# **Xgboost could work in python, but does not work in Anaconda Jupyter notebook**

![3](https://cloud.githubusercontent.com/assets/14320144/19213082/89893ee2-8d6a-11e6-9b86-db56d4f3beab.PNG)

![1](https://cloud.githubusercontent.com/assets/14320144/19213070/6335717a-8d6a-11e6-86a6-c384b31855b6.PNG)
![2](https://cloud.githubusercontent.com/assets/14320144/19213071/654488ac-8d6a-11e6-9a67-b31c6d8c09a3.PNG)
## Environment info

Operating System: Windows 7 64bit

Compiler: 

Package used (python/R/jvm/C++): Python 2.7

`xgboost` version used: 0.6

If you are using python package, please provide
1. The python version and distribution: Python 2.7.12 |Anaconda 4.1.1 (64-bit)

**Comment by wangjiahong666** [comment]
@phunterlau Could you help me with this, please?

**Comment by phunterlau** [comment]
@wangjiahong666 have you verified with this instruction https://www.ibm.com/developerworks/community/blogs/jfp/entry/Installing_XGBoost_For_Anaconda_on_Windows?lang=en and the PATH of your xgboost installation? The PATH can be the tricky key for notebooks.

**Comment by wangjiahong666** [comment]
@phunterlau Thank you
Yes, I verified under the above instruction when I installed Xgboost. 

Now I have figured out how to get Xgboost work in Jupyter Notebook. 
I have to launch Jupyter Notebook by clicking the "Jupyter Notebook" from start menu, then Xgboost works. 
<img width="299" alt="1" src="https://cloud.githubusercontent.com/assets/14320144/19300227/d605ab76-9061-11e6-9169-bacf24643196.png">





If I launch jupyter notebook by Git Bash, then Xgboost does not work. 
Very interesting......
